include/syncmer_hash.hpp

```diff
@@ -24,44 +24,45 @@ namespace seqan3::detail
 {
 //!\brief seqan3::views::syncmer_hash's range adaptor object type (non-closure).
 //!\ingroup search_views
+template <bool open>
 struct syncmer_hash_fn
 {
     /*!\brief Store the kmers and the smers and return a range adaptor closure object.
     * \param[in] kmers       The k-mer size to be used.
     * \param[in] smers       The s-mer size (s<k) to be used.
-    * \throws std::invalid_argument if the window size is smaller than 1.
+    * \throws std::invalid_argument if the k-mer size is smaller than 1.
```

Comment:
Sorry but I think it would be better like this:
* \throws std::invalid_argument if the s-mer size is smaller than 1 or the k-mer size is smaller than the s-mers.